dist/init/linux-systemd/caddy.service

```diff
@@ -30,8 +30,8 @@ LimitNPROC=512
 
 ; Use private /tmp and /var/tmp, which are discarded after caddy stops.
 PrivateTmp=true
-; Use a minimal /dev
-PrivateDevices=true
+; Use a minimal /dev (May bring additional security if switched to 'true', but it may not work on Raspberry Pi's or other devices, so it has been disabled in this dist.)
+PrivateDevices=false
```

Comment:
What if this line (and its associated comment) was just removed altogether?

Reply:
I'm unsure all devices set it to "false" by default. This line forces it.

As it adds security, I think it'll make it's possible use easier if left in that file.